src/components/TaskRunsCard/index.jsx

```diff
@@ -180,15 +180,27 @@ export default class TaskRunsCard extends Component {
     this.setState({ showArtifacts: !this.state.showArtifacts });
   };
 
+  getLiveLogArtifactFromRun = run => {
+    const artifact = run.artifacts.edges.find(({ node: { name } }) =>
+      name.includes('live_backing.log')
```

Comment:
Is it always `live_backing.log`?

Reply:
No, in fact, that's an implementation detail of docker-worker.

https://docs.taskcluster.net/docs/manual/tasks/artifacts#log-artifacts says:
> By convention, workers record the output of a task -- a command's output to stdout and stderr, for example -- in an artifact named public/logs/live.log. The details of this artifact, such as its storage type and encoding, can differ from worker to worker. You should not assume anything about it, as the implementation may change dynamically to optimize performance.
> 
> While a task is running, some workers can relay data to the client as it is produced, in a single HTTP request. Clients can parse and display this data as it arrives to present a "live log" to the user.

So that's the only well-known filename we should be looking for.

It might be worth amending that documentation to suggest (without making any commitments) that log files should be named with `/logs/` in the filename (per https://github.com/taskcluster/taskcluster-web-server/pull/56), or alternately ending in `.log`, as tools will use that as a hint that the content is a log and should be viewed as such.

Reply:
The UI will now only consider a file to be a log when the contentType starts with `text`. https://github.com/taskcluster/taskcluster-web-server/pull/56/commits/4be56ab750601fe120bda28b1d95cf59202ddad5.